src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Dec2HexT_ConsistentOneColumnTableResultDisabled.txt

```diff
@@ -0,0 +1,8 @@
+﻿>> Dec2Hex([{A:2},{A:3}])
+Errors: Error 8-21: Invalid argument type (Table). Expecting a Number value instead.
+
+>> Dec2Hex([{A: 12}])
+Errors: Error 8-17: Invalid argument type (Table). Expecting a Number value instead.
+
+>> Dec2Hex([{A: 12}], 3)
+Errors: Error 8-17: Invalid argument type (Table). Expecting a Number value instead.
```

Comment:
Instead of splitting the test in two files, you can merge them into a single one, and use the Table function instead of the `[...]` notation.